server/src/main/java/org/elasticsearch/ExceptionsHelper.java

```diff
@@ -88,7 +88,7 @@ public static Throwable unwrapCause(Throwable t) {
             }
             if (counter++ > 10) {
                 // dear god, if we got more than 10 levels down, WTF? just bail
-                logger.warn("Exception cause unwrapping ran for 10 levels...", t);
+                logger.warn("Exception cause unwrapping ran for 10 levels... ", t);
```

Comment:
Could you switch this back? It isn't related to your other changes and I don't think it is a change we want anyway.